compiler/prelude/TysWiredIn.hs

```diff
@@ -1072,6 +1101,32 @@ mk_class tycon sc_pred sc_sel_id
   = mkClass (tyConName tycon) (tyConTyVars tycon) [] [sc_pred] [sc_sel_id]
             [] [] (mkAnd []) tycon
 
+{- *********************************************************************
+*                                                                      *
+                Multiplicity Polymorphism
+*                                                                      *
+********************************************************************* -}
+
+-- For information about the usage of the following type,
+-- see Note [TYPE and RuntimeRep] in module TysPrim
```

Comment:
I assume that you have not modified the note. I would suggest adding a few words to be explicit about the fact that multiplicities are handled much like runtime reps.